code/game/turfs/simulated/walls.dm

```diff
@@ -434,3 +443,26 @@
 /turf/simulated/wall/narsie_act()
 	if(prob(20))
 		ChangeTurf(/turf/simulated/wall/cult)
+
+/turf/simulated/wall/proc/add_dent(denttype, x=rand(-8, 8), y=rand(-8, 8))
+	if(LAZYLEN(dent_decals) >= 15)

```

Comment:
Put this in a macro like TG does called MAX_DENT_DECALS please. Avoid magic numbers.

Reply:
fixed